common/model/media-object-list.ts

```diff
@@ -0,0 +1,9 @@
+import { HTMLString } from '../services/prismic/types';
+import { ImageType } from './image';
+
+export type MediaObjectType = {
+  id: string,
+  title: string,
+  text: HTMLString | null,
+  image: ImageType | null,
+};
```

Comment:
Why not just have this file as `media-object`?

Reply:
yes ... i need to rename this .. thanks